tests/unit/phpDocumentor/Renderer/TemplateFactoryTest.php

```diff
@@ -0,0 +1,238 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Renderer {
+
+    use Mockery as m;
+    use phpDocumentor\Renderer\Action\Action1;
```

Comment:
The namespace is fixed because we are testing the version of Actions that are bound to phpDocumentor; I agree with you that the names could then be more descriptive